app/components/common/editing/ConfirmDeleteDialog.tsx

```diff
@@ -0,0 +1,103 @@
+import CloseIcon from '@mui/icons-material/Close';
+import DeleteOutlinedIcon from '@mui/icons-material/DeleteOutlined';
+import Box from '@mui/material/Box';
+import ClickAwayListener from '@mui/material/ClickAwayListener';
+import Dialog from '@mui/material/Dialog';
+import Stack from '@mui/material/Stack';
+import { SxProps } from '@mui/material/styles';
+import Typography from '@mui/material/Typography';
+
+import theme from '@/styles/theme';
+
+import CustomButton from '../CustomButton';
+
+interface ConfirmDeleteDialogProps {
+  open: boolean;
+  onConfirm: () => void;
+  onCancel: () => void;
+}
+
+export const ConfirmDeleteDialog = ({ open, onConfirm, onCancel }: ConfirmDeleteDialogProps) => {
+  return (
+    <Dialog
+      open={open}
+      slotProps={{
+        backdrop: {
+          sx: {
+            backdropFilter: 'blur(5px)',
+            backgroundColor: 'rgba(0, 0, 0, 0.5)',
+          },
+        },
+      }}
+      PaperProps={{
+        sx: {
+          borderRadius: '16px',
+          border: '1px solid rgba(0, 90, 140, 0.20)',
+          background: '#FFFFFF',
+          boxShadow:
+            '0px 4px 26px 3px rgba(0, 0, 0, 0.12), 0px 10px 22px 1px rgba(0, 0, 0, 0.14), 0px 6px 6px -3px rgba(0, 0, 0, 0.05)',
+          width: 'auto',
+        },
+      }}
+    >
+      <ClickAwayListener onClickAway={onCancel}>
+        <Box sx={containerStyle}>
+          <CloseIcon onClick={onCancel} sx={closeIconStyle} />
+          <Typography variant="body2" color="text.primary" sx={textStyle}>
+            Möchtest du diesen Beitrag löschen?
+          </Typography>
+          <Stack direction={'row'} alignItems={'flex-start'} gap={'8px'} flexWrap={'wrap'}>
```

Comment:
Hovering over the buttons in the dialog causes a slight shift - may be related to the border width on hover